prac_06/guitars.py

```diff
@@ -0,0 +1,29 @@
+"""
+CP1404 Practical 6 -
+Make list of user guitars
+Estimate: 10 minutes
+Actually: 12 minutes
+"""
+from guitar import Guitar
+
+CURRENT_YEAR = 2024
+
+guitars = []
+
+print("My guitars!")
+name = input("Name: ")
+while name != "":
+    year = int(input("Year: "))
+    cost = float(input("Cost: $"))
+    guitars.append(Guitar(name, year, cost))
+    print(f"{name} ({year}) : ${cost:.2f} added.")
+    name = input("Name: ")
+
+max_length_name = max([len(guitar.name) for guitar in guitars])
```

Comment:
you did see some magic numbers on your printing, it's definitely a bonus.

Reply:
However, you can research the __lt__ magic method Syntax as a consideration because it is also a built-in, like lambda.